tools/docker/Dockerfile.webkitgtk

```diff
@@ -29,19 +33,11 @@ RUN apt-get -qqy update \
     xvfb \
     git-core
 
-# Configure buster-wpt-webkit-updates repository
-RUN apt-key adv --fetch-keys https://webkitgtk.org/debian/apt.key
-RUN printf 'deb [arch=amd64] https://webkitgtk.org/debian buster-wpt-webkit-updates main' \
-    > /etc/apt/sources.list.d/webkitgtk.list
-RUN printf 'Package: *\nPin: origin webkitgtk.org\nPin-Priority: 1999' \
-    > /etc/apt/preferences.d/99webkitgtk
-
-# Update and upgrade
-RUN apt-get update \
-  && apt-get -y upgrade
 
-# Install webkit packages from https://webkitgtk.org/debian
-RUN apt-get install -y -t buster-wpt-webkit-updates webkit2gtk-driver
+# To speed up testers, install in the image most of WebKitGTK dependencies
+# but don't install WebKitGTK itself (that will be done at test time)
+RUN apt-get install -qqy webkit2gtk-driver \
```

Comment:
Instead of installing and then removing, could we use `apt-get install --download-only` as in https://unix.stackexchange.com/a/408348 to put the packages in /var/cache/apt/archives and make the install faster  in most cases? I think it wouldn't even make the Dockerfile bigger than it is now, because `apt-get remove` probably doesn't remove stuff from the cache.

Reply:
Yes. good idea! :)

Reply:
Pushed a new commit implementing this